Cargo.toml

```diff
@@ -1,37 +1,69 @@
 [package]
 name = "nusb"
-version = "0.1.10"
+version = "0.1.12"
 description = "Cross-platform low-level access to USB devices in pure Rust"
 categories = ["hardware-support"]
 keywords = ["usb", "hardware"]
 authors = ["Kevin Mehall <km@kevinmehall.net>"]
 edition = "2021"
 license = "Apache-2.0 OR MIT"
 repository = "https://github.com/kevinmehall/nusb"
-rust-version = "1.76" # keep in sync with .github/workflows/rust.yml
+rust-version = "1.80"                                                       # keep in sync with .github/workflows/rust.yml
 
 [dependencies]
 atomic-waker = "1.1.2"
 futures-core = "0.3.29"
 log = "0.4.20"
 once_cell = "1.18.0"
 slab = "0.4.9"
+tracing = "0.1"
 
 [dev-dependencies]
 env_logger = "0.10.0"
 futures-lite = "1.13.0"
+pollster = { version = "0.4", features = ["macro"] }
 
-[target.'cfg(any(target_os="linux", target_os="android"))'.dependencies]
-rustix = { version = "0.38.17", features = ["fs", "event", "net"] }
-libc = "0.2.155"
+# [target.'cfg(all(any(target_os="linux", target_os="android"), not(target_arch="wasm32")))'.dependencies]
```

Comment:
Even with the conditional it fails to compile. Not sure how to do this one.

Reply:
Ok consider me confused. Having this last time I tried made it break. It works fine now.